src/lib/functions/synchronous.js

```diff
@@ -36,14 +38,26 @@ const handleSynchronousFunction = function (err, result, response) {
   response.end()
 }
 
-const formatLambdaLocalError = (err) => `${err.errorType}: ${err.errorMessage}\n  ${err.stackTrace.join('\n  ')}`
+const formatLambdaLocalError = (err) =>
+  JSON.stringify({
+    errorType: err.errorType,
+    errorMessage: err.errorMessage,
+    trace: err.stackTrace,
+  })
 
 const handleErr = function (err, response) {
+  const errorTemplateFile = fs.readFileSync(pathModule.join(__dirname, './templates/function-error.html'), 'utf-8')
```

Comment:
[boulder] I think we should try to read the template once and keep it in memory, as opposed to reading it from disk each time we process an error.

Reply:
Does [this block](https://github.com/netlify/cli/pull/4486/files#diff-a533fc859ae76f3974782aff19d74d697ff83bf2cec5c6ce8f04f742d5946f93R48-R53) suffice? Moved it out so it's only called when we need to render the template.